frontend/lib/app.dart

```diff
@@ -45,14 +51,43 @@ class App extends StatelessWidget {
             ? buildConfig.projectId.local
             : buildConfig.projectId.showcase;
 
-    final routes = <String, WidgetBuilder>{
-      introRouteName: (context) => IntroScreen(
-            onFinishedCallback: () => settings.setFirstStart(enabled: false),
-          ),
-      homeRouteName: (context) => const HomePage()
-    };
-
-    final String initialRoute = settings.firstStart ? introRouteName : homeRouteName;
+    final GoRouter router = GoRouter(
+      routes: <RouteBase>[
+        GoRoute(
+          path: initialRouteName,
+          builder: (BuildContext context, GoRouterState state) {
+            return settings.firstStart
+                ? IntroScreen(
+                    onFinishedCallback: () => settings.setFirstStart(enabled: false),
+                  )
+                : HomePage();
+          },
+          routes: [
+            GoRoute(
+              path: '$activationRouteName/:$activationRouteCodeParamName',
```

Comment:
I think you should decide whether you want to use /activation/code or /activation/base64qrcode
```suggestion
              path: '$activationRouteName/$activationRouteCodeParamName',
```

Reply:
if i remove the `:` the route will not be resolved correctly and the fragment will not be read. 
The  route schema is `activation/code#<base64code>`
no idea whats your point here